Valet.podspec

```diff
@@ -14,18 +14,4 @@ Pod::Spec.new do |s|
   s.macos.deployment_target = '10.11'
 
   s.tvos.exclude_files = 'Sources/SinglePromptSecureEnclaveValet.swift'
-
```

Comment:
cc @dnkoutso. I'm pretty darned sure that the `test_spec` can't handle test environments that require the Keychain Sharing entitlement.

Reply:
Are there any side effects of removing the test spec?

Reply:
if its urgent you can remove this now and I can re-enable.

Reply:
This is indeed urgent. Thank you!